EmojiTextArea/pom.xml

```diff
@@ -23,14 +23,6 @@
                 <configuration>
                     <release>11</release>
                     <compilerArgs>
-                        <arg>--add-opens</arg>
```

Comment:
did you change RichTextFX dependency/version?

Reply:
Yes, please check [here](https://github.com/gluonhq/wave-app/pull/200/files#diff-d91e51b39d00c9f8d4b3ac0f34c5fb717e99452165a8fff208131ee6b13beec2R92)